packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -109,6 +137,46 @@ contract ControllerHelper is FlashControllerHelper, IERC721Receiver {
         emit FlashWMint(msg.sender, _vaultId, _wPowerPerpAmount, amountToFlashswap, _collateralAmount);
     }
 
+    /**
+     * @notice flash close position and buy long squeeth
+     * @param _vaultId vault ID
+     * @param _wPowerPerpAmount amount of WPowerPerp to burn
+     * @param _collateralToWithdraw amount of collateral to withdraw
+     * @param _minToReceive minimum amount of long WPowerPerp to receive
+     */
+    function flashWBurnBuyLong(
+        uint256 _vaultId,
+        uint256 _wPowerPerpAmount,
+        uint256 _collateralToWithdraw,
+        uint256 _minToReceive
```

Comment:
think this should be an exactOut swap (so wPowerPerpAmount to burn + wPowerPerpAmount to go long)

think should have a maxToPay param